hera_cal/frf.py

```diff
@@ -16,11 +16,85 @@
 from pyuvdata import UVData, UVFlag
 import argparse
 from . import io
+from . import vis_clean
 import warnings
+import astropy.constants as const
 
 
-def timeavg_waterfall(data, Navg, flags=None, nsamples=None, wgt_by_nsample=True, 
-                      wgt_by_favg_nsample=False, rephase=False, lsts=None, freqs=None, 
+SPEED_OF_LIGHT = const.c.si.value
+SDAY_KSEC = 86163.93 / 1000.
+
+
+def sky_frates(uvd, keys=None, frate_standoff=0.0, frate_width_multiplier=1.0, min_frate_half_width=0.025):
+    """Automatically compute sky fringe-rate ranges based on baselines and telescope location.
+
+    Parameters
+    ----------
+    uvd: UVData object
+        uvdata object of data to compute sky-frate limits for.
+    keys: list of antpairpol tuples, optional
+        list of antpairpols to generate sky fringe-rate centers and widths for.
+        Default is None -> use all keys in self.data.
+    frate_standoff: float, optional
+        Additional fringe-rate standoff in mHz to add to Omega_E b_{EW} nu/c for fringe-rate inpainting.
+        default = 0.0.
+    frate_width_multiplier: float, optional
+        fraction of horizon to fringe-rate filter.
+        default is 1.0
+    min_frate_half_width: float, optional
```

Comment:
other funcs call this variable `min_frate`, shall we use this instead? its not really a minimum half_width, its just a minimum frate, right?

Reply:
It is actually a half width. I'm going to change the other functions to agree with this.

Reply:
okay, but the description still reads "minimum frate to filter". does this assume the filter is centered at 0 mHz and is symmetric? Perhaps this can be a *little* more verbose to explain, thanks. also if this works similar to min-frate in other funcs, its not the min-frate to filter (which admittedly is a confusing term), but the minimum bounds on the extent of any fr-filter (i.e. all fr-filters will model out to min_frate or greater)

Reply:
I've updated this docstring (and other equivalent docstrings for the same arg in other functions) to clarify that this is the minimum half width of the fringe-rate filter, not literally the minimum fringe-rate that will be filtered (equivalent to min_dly arg in the delay_filters).